### ISSUE TITLE
Standardize all functions and claims that change with coordinate handedness

### ISSUE THREAD

**Issue by quchen** [quchen/generative-art#21]
tl;dr
====

Standardize all functions and documentation to standard math coordinates

Long description
=======

Cairo has zero at the top-left, and the documentation often reflects that. For the most part, I simply winged it, and fiddled with the signs long enough so the picture is pretty. I never checked whether _clockwise_ is actually clockwise for example, and even worse: now that we have both a Cairo and a penplotting backend, claims that change with coordinate handedness are wrong for one of the backends.

I think it’s therefore a good idea to check the functions that rotate things for whether their documentation is right, and fixing any sign errors (and un-fixing those sign errors so output of previous pictures does not change).
